packages/@react-aria/visually-hidden/src/VisuallyHidden.tsx

```diff
@@ -24,33 +24,57 @@ interface VisuallyHiddenProps extends HTMLAttributes<HTMLElement> {
    */
   elementType?: string | JSXElementConstructor<any>,
 
-  /** Whether the content can be focused. */
+  /** Whether the element should become visible on focus, for example skip links. */
   isFocusable?: boolean
 }
 
+const styles: CSSProperties = {
+  border: 0,
+  clip: 'rect(0 0 0 0)',
+  clipPath: 'inset(50%)',
+  height: 1,
+  margin: '0 -1px -1px 0',
+  overflow: 'hidden',
+  padding: 0,
+  position: 'absolute',
+  width: 1,
+  whiteSpace: 'nowrap'
+};
+
 /**
  * VisuallyHidden hides its children visually, while keeping content visible
  * to screen readers.
  */
 export function VisuallyHidden(props: VisuallyHiddenProps) {
   let {
     children,
-    className,
+    style,
     elementType: Element = 'div',
     isFocusable,
     ...otherProps
   } = props;
 
-  className = clsx(
-    styles['u-react-spectrum-screenReaderOnly'],
-    {[styles['is-focusable']]: isFocusable},
-    className
-  );
+  let [isFocused, setFocused] = useState(false);
+  let {focusProps} = useFocus({
```

Comment:
should useFocus return isFocused as well, like the useHover hook?
https://github.com/adobe/react-spectrum/blob/452d1cb6a49f9f493757737edaf2b64014108de6/packages/%40react-aria/interactions/src/useHover.ts#L185

Reply:
ehh maybe a separate task. Not sure whether we want to add state to every call to `useFocus` though